index.es6.js

```diff
@@ -69,5 +73,31 @@ if (window.history.pushState) {
     }
   };
 
+  // Calculating width of browser's scrollbar
+  function getScrollbarWidth() {
+    let outer = document.createElement("div");
+    outer.style.visibility = "hidden";
+    outer.style.width = "100px";
+    outer.style.msOverflowStyle = "scrollbar";
```

Comment:
Just a thought here, but would it make sense to make this `position: absolute; left: -400px` or something to force render off-screen, preventing a "jumpy page" while the temporary div is present?